src/Http/Headers/src/HeaderUtilities.cs

```diff
@@ -562,6 +562,11 @@ public unsafe static string FormatNonNegativeInt64(long value)
                 throw new ArgumentOutOfRangeException(nameof(value), value, "The value to be formatted must be non-negative.");
             }
 
+            if (value == 0)
```

Comment:
Certified Z E S T Y

Reply:
If it is stupid and it works, is it really stupid? 🐴

Reply:
𝓟𝓻𝓸𝓯𝓸𝓾𝓷𝓭